src/driver.c

```diff
@@ -248,7 +247,7 @@ TegraProbeHardware(const char *dev, struct xf86_platform_device *platform_dev)
 #endif
 
     fd = TegraOpenHardware(dev);
-    if (fd != -1) {
+    if (fd > 0) {
```

Comment:
Not that this really matters, but I think `fd >= 0` is better to be consistent with the `fd < 0` test above...